pkg/sql/bulkmerge/merge_processor.go

```diff
@@ -156,8 +165,117 @@ func (m *bulkMergeProcessor) handleRow(row rowenc.EncDatumRow) (rowenc.EncDatumR
 
 // Start implements execinfra.RowSource.
 func (m *bulkMergeProcessor) Start(ctx context.Context) {
-	m.StartInternal(ctx, "bulkMergeProcessor")
-	m.input.Start(ctx)
+	ctx = m.StartInternal(ctx, "bulkMergeProcessor")
+	// TODO(annie): also find a way to process input/get task from coordinator
+	if err := m.mergeSSTs(ctx, 0); err != nil {
```

Comment:
The mergeSST call should be in handleRow. handleRow is called once for each task. Start is called once when the flow is set up.